src/main/java/com/google/gfx/services/order/OrderService.java

```diff
@@ -0,0 +1,85 @@
+package com.google.gfx.services.order;
+
+import com.google.gfx.model.Order;
+import com.google.gfx.model.User;
+
+import java.util.Collection;
+import java.util.Currency;
+
+/**
+ * Manages the orders in the system.
+ */
+public interface OrderService {
+
+  /**
+   * Stores an order.
+   */
+  void storeOrder(Order order);
```

Comment:
storeOrder might be preferable since the order is created when you construct the Order object?